events/events.go

```diff
@@ -39,7 +39,9 @@ type Stream interface {
 	Open() error
 
 	// Publish publishes the message to the message broker.
-	Publish(ctx context.Context, subject string, msg []byte) error
+	//
+	// rollupSubject when set to true will cause any previous messages with the same subject to be overwritten by this new msg.
+	Publish(ctx context.Context, subject string, msg []byte, rollupSubject bool) error
```

Comment:
This `rollupSubject` can cause problems on out-of-band operations right? It will clobber the contents of any subject on a stream where `AllowRollup` is set. Maybe it's better and safer to create a different stream for in-band operations?

Reply:
Currently the subject for our stream messages are in the form `$prefix.$condition`, and our controllers listen on `$prefix.$condition`.

I intend to update the subject for all our stream messages to `$prefix.$condition.$serverID`,
 - The out-of-band controllers are to listen on `$prefix.$condition.*`
 - The in-band controllers are to listen on `$prefix.$condition.$serverID`
 
The subject change along with `AllowRollup` ensures that only a single message for a condition, server can be present on the stream for server which is what we want for server Conditions